crates/re_viewer_context/src/gpu_bridge/tensor_to_gpu.rs

```diff
@@ -465,7 +465,12 @@ fn pad_and_narrow_and_cast<T: Copy + Pod>(
 fn height_width_depth(tensor: &Tensor) -> anyhow::Result<[u32; 3]> {
     use anyhow::Context as _;
 
-    let shape = &tensor.shape();
+    let mut shape = tensor.shape_short();
+
+    // Handle 1x1 and 1x1x1 images.
+    if shape.len() == 1 {
+        shape = tensor.shape();
+    }
```

Comment:
Won't this fail for `WxHx1x1` ?

Why not use `tensor.image_height_width_channels` here?

Reply:
WxHx1x1 would get the short shape of WxH which should work. Anyways, I didn't want to kill most of your nice errors here, but now that you called it I'll do that instead :)